### ISSUE TITLE
Documentation for loading config in bundles produces a Psalm v5 error ParamNameMismatch

### ISSUE THREAD

**Issue by dkarlovi** [symfony/symfony-docs#17795]
```
ERROR: ParamNameMismatch - src/DependencyInjection/MyExtension.php:20:59 - Argument 2 of MyExtension::load has wrong name $containerBuilder, expecting $container as defined by Symfony\Component\DependencyInjection\Extension\ExtensionInterface::load (see https://psalm.dev/230)
    public function load(array $configs, ContainerBuilder $containerBuilder): void
```

Example given [here](https://symfony.com/doc/current/bundles/extension.html#creating-an-extension-class):

```php
    public function load(array $configs, ContainerBuilder $containerBuilder)
    {
        // ... you'll load the files here later
    }
```

The param is [currently indeed called `$container`](https://github.com/symfony/symfony/blob/395eb9a43d3baac20bf959e8523fccf7e34c85cd/src/Symfony/Component/DependencyInjection/Extension/ExtensionInterface.php#L30).

**Comment by dkarlovi** [comment]
I guess this was done on purpose in #17683, but it does have a consequence of "default errors" for Psalm users, it might be worth reviewing.

**Comment by javiereguiluz** [comment]
Good catch. Hopefully we can change the variable name in the code. I think it's misleading because a "container builder" variable is not a container. IT could be just a builder, but we can make it less generic and call it "container builder"

**Comment by xabbuh** [comment]
Fixed by #18299?

**Comment by javiereguiluz** [comment]
Yes, let's close this as fixed after #18299 was merged. Thanks.
